### PR TITLE
Doc changes for 1.0.0

### PR DESCRIPTION
**Pull request by nannanli**

### CODE REVIEW

**Comment by JamesBauman** [comment]
@nannanli  - Do you want me to merge this PR?

**Comment by nannanli** [comment]
Hi Jim @JamesBauman and all, 
Please keep this PR open until the 1.0 publish date. Let's leave it to Jack to click the Merge button once the 1.0 build is published. :-)

File: docs/user-guide/install-zos.md [hunk 1]
```diff
@@ -519,93 +537,217 @@ You can obtain the _asid_ from the value of `A=asid` when you issue the followin
...
 
     ```
-    zowe-verify.sh
+    PPT PGMNAME(ZWESIS01) KEY(4) NOSWAP
     ```
+    After you edit the PARMLIB, issue the following command to make the SCHEDxx changes effective:
 
-The script writes its messages to your terminal window.  The results are marked `OK`, `Info`, `Warning` or `Error`.  Correct any reported errors and restart the Zowe server.  The `zowe-verify.sh` script does not change any settings, so you can run it as often as required.
+    ```
+    /S SET SCH=xx
```
Comment (MarkAckert) [inline docs/user-guide/install-zos.md]: Convert to : 
```suggestion
    /S SET SCH=xx
```
/SET SCH=xx

/S SET SCH=xx does not work on my system, /SET SCH=xx does . Any comment from z/OS experts?

Comment (MarkAckert) [reply docs/user-guide/install-zos.md]: @Joe-Winchester @dmitrynikolaev @ifakhrutdinov

Comment (jackjia-ibm) [reply docs/user-guide/install-zos.md]: @MarkAckert Nayer is working on a fix of this page: https://github.com/NayerNajafi/docs-site/blob/docs-staging/docs/user-guide/install-zos.md#manually-installing-the-zowe-cross-memory-server

File: docs/user-guide/install-zos.md [hunk 2]
```diff
@@ -519,93 +537,217 @@ You can obtain the _asid_ from the value of `A=asid` when you issue the followin
...
+    D PROG,APF,DSNAME=ZWES.SISLOAD
+    ```
+    where the value of DSNAME is the name of the data set that contains the ZWESIS01 load module.
 
-### Verifying Zowe Application Framework installation
+    To dynamically add the SMS-managed library to the APF list, run the following TSO command:
 
-If the Zowe Application Framework is installed correctly, you can open the Zowe Desktop from a supported browser.
+    ```
+    SETPROF APF,ADD,DSNAME=ZWES.SISLOAD
```
Comment (MarkAckert) [inline docs/user-guide/install-zos.md]:
```suggestion
    SETPROG APF,ADD,DSNAME=ZWES.SISLOAD,VOLUME=YOUR_VOL
```

Comment (MarkAckert) [reply docs/user-guide/install-zos.md]: @dmitrynikolaev @ifakhrutdinov @Joe-Winchester @John-A-Davies

File: docs/user-guide/install-zos.md (hunk 1 above)
Comment (nannanli) [reply docs/user-guide/install-zos.md]: This is fixed now in doc. Thanks Nayer and Mark!

**Pull request merged**